File: task-03/Elixir/SubTask_3.ex
IO.puts "Enter the number n: "
input=IO.gets("") |> String.trim()
num=String.to_integer(input)
n=if rem(num,2)==0 do
  num |> div(2)
else
  (num+1) |> div(2)
end
for i<-1..n do
  IO.puts(String.duplicate(" ",n-i) <> String.duplicate("*",2*i-1))
end
for i<-n-1..1 do
  IO.puts(String.duplicate(" ",n-i) <> String.duplicate("*",2*i-1))
end

File: task-03/Ruby/SubTask_3.rb
print "Enter the number n: "
n=gets.chomp.to_i
n=(n/2.0).ceil
for i in 1..n
  for y in 1..(n-i)
    print " "
  end
  for x in 1..(2*i-1)
    print "*"
  end
  puts
end
for i in (n-1).downto(1)
  for y in 1..(n-i)
    print " "
  end
  for x in 1..(2*i-1)
    print "*"
  end
  puts
end
